ui/spells/spells.pug

```diff
@@ -19,18 +19,18 @@
     = '\n'
     = '\n'
     | &emsp; &emsp;
-    .spell-header-action-area
-      .label.wide.larger Spell Hit
+    .spell-header-action-area.flexing
```

Comment:
I want to move away from these abstract class names, since they make it hard to reason about field styles.
Instead, let's update the specific local scss files.

For example, flexing is unclear, and could mean flex 1, or maybe display flex, or flex auto.

Reply:
I see your point!
Is right thing for me to do to add a new class in spells.pug for this kind of flexing?

Reply:
Yes, I'd add to the section in the spell scss file,  as needed.
I'd also use multiples of 8 px

Reply:
How's this?